third_party/java/proguard/BUILD

```diff
@@ -15,3 +15,24 @@ filegroup(
     ],
     visibility = ["//src:__pkg__"],
 )
+
+java_import(
+    name = "proguard_import",
+    jars = ["proguard5.3.3/lib/proguard.jar"],
+)
+
+java_binary(
+    name = "proguard",
+    main_class = "proguard.ProGuard",
+    runtime_deps = [":proguard_import"],
+)
+
+# To be bundled into remote Java tools
+filegroup(
```

Comment:
I also took a look at #8153. The way you're doing it now will place the proguard jar at the top level directory of the java_tools zip. Since this is third_party I would have it in java_tools also under `third_party` (like the jacoco agent see #8137). You can create here a zip with proguard like this:

```
genrule(
    name = "proguard_zip",
    srcs = ["proguard5.3.3/docs/GPL.html", ":proguard_deploy.jar",],
    outs = ["proguard.zip"],
    cmd = "$(location //src:zip_files) third_party/java/proguard $@ $(SRCS)",
    tools = ["//src:zip_files"],
    visibility = ["//src:__pkg__"],
)

```

Afterwards you can add this zip as a dependecy to `//src:java_tools_java*_no_build_zip`